src/ziggurat/init.clj

```diff
@@ -76,7 +80,9 @@
   (start-rabbitmq-consumers args))
 
 (defn- stop-rabbitmq-connection []
-  (mount/stop #'connection))
+  (mount/stop #'cpool/channel-pool)
```

Comment:
can replace line 83-85 with mount/only since it will determine the order according to namespace dependency graph